### DISCUSSION TITLE
7 inch LCD touchscreen (more a question than an issue)

### DISCUSSION THREAD

**Discussion by AL1-1956** [onlaj/Piano-LED-Visualizer#423] [General]
Hello
This is not an issue but more a question 
I bought recently this LCD Screen
https://www.amazon.fr/dp/B087GFY4KL?psc=1&ref=ppx_yo2ov_dt_b_product_details
It's a 5 touch points LCD screen
Do You think that it'is possible to use it as a display with touch features for the Piano Visualizer ?
Anyway
Have a fine New Year's Eve
Alain

**Comment by onlaj** [comment]
Hello.
I'm not familiar with that specific LCD panel, but from what I can see on store page it can display the system just like a regular monitor. You can just run any browser and open Visualizer's web interface.

**Comment by AL1-1956** [comment]
Okay thank you very much for the quick answer
Now I have to learn how to use a touchscreen with a Pi
(learning becomes difficult when you are 67 years old)

Alain

**Comment by onlaj** [comment]
It's never too late to learn new things :)
If you're using Visualizer image from releases you'll have to install some GUI, by default it's using terminal-only. If you are installing everything from scratch you can use full Raspberry Pi os instead of Lite, it will have GUI installed by default.
